### PR TITLE
Feature/update entity

### PR DESCRIPTION
**Pull request by dsmith42**
[x] - Updates the `Entity.Meal` class:
[x] - Reflect changes to `db.json` file
[x] - Update ContentView to display all information in iOS app
[x] - Update git ignore file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,15 +1,119 @@
...
+# CocoaPods
+#
+# We recommend against adding the Pods directory to your .gitignore. However
+# you should judge for yourself, the pros and cons are mentioned at:
+# https://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control
+#
+# Pods/
+#
+# Add this line if you want to avoid checking in source code from the Xcode workspace
+# *.xcworkspace
```
Comment (Gmiller290488) [inline .gitignore:119]: 😍

File: iosHackthonApp/iosHackthonApp/ContentView.swift [hunk 2]
```diff
@@ -69,10 +69,10 @@ struct MealRow: View {
                 Text("Name: \(meal.name)")
                 Text("Description: \(meal.info)")
                 Text("Temperature: \(meal.hot ? "Hot" : "Cold")")
-                Text("Available from: \(meal.availableFrom)")
+                Text("Available from: \(meal.availableFromDate)")
                 Text("Expires: \(meal.expiryDate)")
-
-
+                Text("Lat, Lon: [\(meal.locationLat), \(meal.locationLong)]")
+                Text("Pick up code test: \(String(meal.id.suffix(4)))")
```
Comment (Gmiller290488) [inline iosHackthonApp/iosHackthonApp/ContentView.swift:75]: 👌

**Review by Gmiller290488 — APPROVED** [review]
Looks good!

File: .gitignore (hunk 1 above)
Comment (dsmith42) [reply .gitignore:119]: Just noticed it was still commented out, should we uncomment it now?

Comment (Gmiller290488) [reply .gitignore:119]: That emoji was just aimed at the file in general. I think it's fine

**Pull request merged**